frontend/public/components/utils/_list-input.scss

```diff
@@ -1,6 +1,9 @@
 .co-list-input__add-btn {
   padding-left: 0;
 }
+.co-list-input__help-block {
```

Comment:
style nit: we should try to have consistent newlines between rules (so newline above this rule)

Reply:
@spadgett fixed space and squashed.